tensorflow/lite/c/builtin_op_data.h

```diff
@@ -71,6 +71,7 @@ typedef enum {
   kTfLiteActTanh,
   kTfLiteActSignBit,
   kTfLiteActSigmoid,
+  kTfLiteActRelu0To1,  // min(max(0, x), 1)
 } TfLiteFusedActivation;
```

Comment:
is TfLiteFusedActivation persisted in file like tflite file? If yes, then I agree with putting it at the bottom (out of order with other kTfLiteActReluXX) to maintain backward compatibility. 

overall LGTM.

Reply:
Yes, in the internal CL, it also adds `kTfLiteActRelu0To` at the bottom for backward compatibility